openmetadata-docs/content/v1.7.x-SNAPSHOT/getting-started/day-1/hybrid-saas/hybrid-ingestion-runner.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Hybrid Ingestion Runner
+slug: /getting-started/day-1/hybrid-saas/hybrid-ingestion-runner.md
+collate: true
+---
+
+# Hybrid Ingestion Runner
+
+The **Hybrid Ingestion Runner** is a component designed to enable Collate customers operating in hybrid environments to securely execute ingestion workflows within their own cloud infrastructure. In this setup, your SaaS instance is hosted on Collate’s cloud, while tools like Argo Workflows are deployed within your private cloud. The Hybrid Runner acts as a bridge between these two environments, allowing ingestion workflows to be triggered and managed remotely—without requiring the customer to share secrets or sensitive credentials with Collate. It securely receives workflow execution requests and orchestrates them locally, maintaining full control and data privacy within the customer’s environment.
+
+## Prerequisites
+
+Before setting up the Hybrid Ingestion Runner, ensure the following:
+
+- Hybrid Runner has been setup
+- Secrets manager configured on your cloud.
+
+## Configuration Steps for Admins
+
+Once your DevOps team has installed and configured the Hybrid Runner, follow these steps as a Collate Admin to configure services and manage ingestion workflows.
+
+### 1. Validate Hybrid Runner Setup
+
+- Go to **Settings > Preferences > Ingestion Runners** in the Collate UI.
+- Look for your runner in the list.
+- The status should display as **Connected**.
+
+> If the runner is not connected, reach out to Collate support.
+
+{% image
+src="/images/v1.7/getting-started/ingestion-runner-preferences.png"
+/%}
+
+{% image
+src="/images/v1.7/getting-started/ingestion-runner-list.png"
+/%}
+
+### 2. Create a New Service
+
+- Navigate to **Settings > Services**.
+- Click **+ Add New Service**.
+- Fill in the service details.
+- In the “Ingestion Runner” dropdown, choose the hybrid runner.
+
+{% image
+src="/images/v1.7/getting-started/ingestion-runner-service.png"
+/%}
+
+> Choose "Collate SaaS Runner" to run ingestion workflows within Collate’s SaaS environment, even if you're operating in hybrid mode.
+
+### 3. Manage Secrets Securely
+
+Use your existing cloud provider's Secrets Manager to store sensitive credentials (like usernames and passwords), and reference them securely in Collate via the Hybrid Runner.
```

Comment:
Let's add a line here saying "When executing workflows on your Hybrid environment, you have to use your existing cloud provider's Secrets Manager,...."